eap6-support/hawkular-javaagent-wildfly-feature-pack-eap6/src/main/resources/featurepack/content/standalone/configuration/hawkular-javaagent-config.yaml

```diff
@@ -750,6 +750,11 @@ resource-type-set-dmr:
     - name: Base Directory
       attribute: base-dir
       path:      /core-service=server-environment
+    operation-dmr:
+    - name: JDR
+      internal-name: generate-jdr-report
+      modifies:      true
```

Comment:
modifies=true should probably be removed here as well as in the other place in hawkular-wildfly-agent-feature-pack/src/main/resources/subsystem-templates/hawkular-wildfly-agent.xml.

Note that you do not have modifies in the subsystem XML config (so the two configs - for javaagent and subsystem agent - are different now).

modifies=true means the operation will change something in the managed resource (perhaps a configuration setting or some runtime behavior). When this is the case, the agent will disallow this operation from running when run in a container or when it is "immutable".

Asking for a JDR report (AFAIU) is simply a read-only function and returns a debug report. If that is true, "modifies" should probably be false to allow someone to ask for a JDR report even if the agent is running in immutable or in-container mode. Otherwise, this modifies=true will not allow anyone to get debug information via JDR when running  while immutable/in-container.